code/_macros.dm

```diff
@@ -78,4 +78,7 @@
 #define QDEL_NULL(x) if(x) { qdel(x) ; x = null }
 
 // Spawns multiple objects of the same type
-#define cast_new(type, num, args...) if((num) == 1) { new type(args) } else { for(var/i=0;i<(num),i++) { new type(args) } }
\ No newline at end of file
+#define cast_new(type, num, args...) if((num) == 1) { new type(args) } else { for(var/i=0;i<(num),i++) { new type(args) } }
+
+//Makes span tags easier
+#define span(class, text) ("<span class='[class]'>[text]</span>")
```

Comment:
A span macro of my own design, takes a class of your choice.
Not mutually exclusive with eris' existing SPAN_WARNING or similar